portfolio/src/main/webapp/script.js

```diff
@@ -17,8 +17,9 @@ function getComments() {
 
   document.getElementById('greeting-container').innerHTML = "";
   var commentNum = parseInt(document.getElementById('quantity').value);
+  var languageSetting = document.getElementById('language').value;
   
-  fetch('/comments?quantity=' + commentNum).then(response => response.json()).then((posts) => {
+  fetch('/comments?quantity=' + commentNum + '&language=' + languageSetting).then(response => response.json()).then((posts) => {
```

Comment:
Can you make the literals in this path constants?

Reply:
yes!